lib/src/Pos/Util/UserSecret.hs

```diff
@@ -93,11 +93,17 @@ deriving instance Eq EncryptedSecretKey => Eq WalletUserSecret
 
 makeLenses ''WalletUserSecret
 
+fixedNM :: NetworkMagic
+fixedNM = NetworkMainOrStage
+
 instance Buildable WalletUserSecret where
     build WalletUserSecret{..} =
         bprint ("{ root = "%addressF%", set name = "%build%
                 ", wallets = "%pairsF%", accounts = "%pairsF%" }")
-        (makeRootPubKeyAddress $ encToPublic _wusRootKey)
+        -- TODO mhueschen |
+        -- TODO @intricate: Will probably have to add NetworkMagic to WalletUserSecret
+        -- instead of using NetworkMainOrStage here :/
+        (makeRootPubKeyAddress fixedNM $ encToPublic _wusRootKey)
```

Comment:
Can these comments be deleted? I forget what the resolution was, but we should avoid `TODO`'s in production code.

We should either make the fix here, or record a TODO somewhere else (e.g. YouTrack) and make the fix here later.

Reply:
It look like we didn't resolve this on `release/1.3.1` so I'll resolve it here.